src/client/user/components/UserLinkTable/ToolBar/SearchInput/index.tsx

```diff
@@ -138,11 +140,10 @@ const SearchInput = () => {
       applySearch()
     }, 500)
     return () => clearTimeout(timeoutId)
-  }, [query])
+  }, [searchText])
```

Comment:
note: not entirely sure that this effect should be placed here, but idk how we can do better - any thoughts? `searchText` belongs to the global redux store, and yet this effect for applying the search resides within this local component. this setup may be confusing because changes made to `searchText` anywhere else from any other component, e.g. `EnhancedTableBody`, will trigger this effect here as well (so long as this component exists).